jaguar_client/lib/src/client.dart

```diff
@@ -287,7 +287,7 @@ class JsonClient {
 
     if (manageCookie) route.interceptBefore(jar.intercept);
```

Comment:
must be only jar right ?

Reply:
Yes. I have not started updating other packages yet. Only working on `resty`.

Reply:
ho ok ^^ all good then :)